apps/settings/src/views/Apps.vue

```diff
@@ -396,6 +396,18 @@ export default {
 	right: -8px;
 }
 
+// Fix wrong color for active icons
+[data-themes*=dark] {
+	:deep(.app-navigation-entry.active .app-navigation-entry-icon) {
+		filter: var(--primary-invert-if-bright);
+	}
+}
+[data-themes*=light] {
+	:deep(.app-navigation-entry.active .app-navigation-entry-icon) {
+		filter: var(--primary-invert-if-dark);
+	}
+}
```

Comment:
Here you go @szaimen 

```suggestion
// Fix wrong color for active icons
@media (prefers-color-scheme: dark) { body {
	.app-navigation-entry.active .app-navigation-entry-icon {
		filter: var(--primary-invert-if-bright);
	}
}}

@media (prefers-color-scheme: light) { body {
	.app-navigation-entry.active .app-navigation-entry-icon {
		filter: var(--primary-invert-if-dark);
	}
}}

[data-themes*=dark] {
	.app-navigation-entry.active .app-navigation-entry-icon {
		filter: var(--primary-invert-if-bright);
	}
}
[data-themes*=light] {
	.app-navigation-entry.active .app-navigation-entry-icon {
		filter: var(--primary-invert-if-dark);
	}
}

```

Reply:
Thanks @skjnldsv ! :)

Reply:
I just tested this and it doesnt seem to work. Maybe gecause the scss is scoped? :/

Reply:
Yes, you need to keep the `:deep(`, I tested it live, locally, so it worked for me.

Reply:
I see. Where exactly would I need to put the :deep( ?

Reply:
Like it was before

Reply:
you mean like that?
![image](https://github.com/nextcloud/server/assets/42591237/0766b6e3-1c87-4eb9-89f3-dc50ed7aa095)

Reply:
This doesnt seem to make it work unfortunately 😢